lua/claudecode/config.lua

```diff
@@ -76,9 +84,19 @@ function M.apply(user_config)
     config = vim.tbl_deep_extend("force", config, user_config)
   end
 
+  -- Check environment variable for terminal_cmd if not set in user config
+  if config.terminal_cmd == nil then
+    local env_cmd = vim.fn.getenv("CLAUDE_TERMINAL_CMD")
+    if env_cmd ~= vim.NIL then
+      config.terminal_cmd = env_cmd
+    end
+  end
```

Comment:
I would rather have the user configure that in their settings and then pass that value into the terminal_cmd instead of us doing it here.